panels/site_sections/panel_app_management.py

```diff
@@ -231,3 +231,26 @@ def everything(self, out, session):
                 ' / '.join(app.tech_needs_labels) + (' / ' if app.other_tech_needs else '') + app.other_tech_needs,
                 app.applied.strftime('%Y-%m-%d')
             ] + panelists)
+
+    def panel_poc_schedule(self, session, attendee_id):
+        attendee = session.attendee(attendee_id)
+        event_times = defaultdict(lambda: defaultdict(lambda: (1, '')))
+        for pa in attendee.panel_applications:
+            if pa.event is not None:
+                for timeslot in pa.event.half_hours:
```

Comment:
I like @kitsuta's suggestion to change the ``pa`` variable name to ``app``, which is still a short enough variable name to not be confusing but is more descriptive - can you make that change?